portfolio/src/main/webapp/script.js

```diff
@@ -66,3 +63,46 @@ function createListElement(text) {
   liElement.classList.add('list-group-item');
   return liElement;
 }
+
+function showAbout() {
+  var elem1 = document.getElementById("about");
+  var elem2 = document.getElementById("skills");
+  var elem3 = document.getElementById("contact");
+
+  if (elem1.style.display === "none") {
+    elem1.style.display = "block";
+    elem2.style.display = "none";
+    elem3.style.display = "none";
+  } else {
+    elem1.style.display = "none";
+  }
+}
+
+function showSkills() {
+  var elem1 = document.getElementById("skills");
+  var elem2 = document.getElementById("about");
+  var elem3 = document.getElementById("contact");
+
+  if (elem1.style.display === "none") {
+    elem1.style.display = "block";
+    elem2.style.display = "none";
+    elem3.style.display = "none";
+  } else {
+    elem1.style.display = "none";
+  }
+}
+
+function showContact() {
+  var elem1 = document.getElementById("contact");
+  var elem2 = document.getElementById("skills");
+  var elem3 = document.getElementById("about");
+
+  if (elem1.style.display === "none") {
+    elem1.style.display = "block";
+    elem2.style.display = "none";
+    elem3.style.display = "none";
+  } else {
+    elem1.style.display = "none";
+  }
+
+}
```

Comment:
You're duplicating code here, you've 2 options:

1. Write a function that hides all the elements and re-use it while showing the correct element afterwards.

2. Write a function with a parameter that specifies which element to display, then you can call it from the HTML directly by binding it to the element by doing, for example: ``onClick="showSection('contact')"``

Reply:
Solved